packages/fast-tooling-react/src/css-editor/color/color.tsx

```diff
@@ -26,9 +26,6 @@ export default class CSSColor extends Foundation<
     public render(): React.ReactNode {
         return (
             <div className={get(this.props, "managedClasses.cssColor")}>
-                <label className={get(this.props, "managedClasses.cssColor_label")}>
-                    Color
-                </label>
                 <div className={get(this.props, "managedClasses.cssColor_control")}>
```

Comment:
i don't see a need for this extra div

Reply:
IF we remove the extra div we no longer need the `cssColor_control` className should we remove that from the contract?

Reply:
Yes, we can put this feature in the tooling v2 branch so it's alright to make all of the breaking changes we need. Be sure to add them to this document in the wiki: https://github.com/microsoft/fast-dna/wiki/Tooling:-Migration-to-Version-2